web/src/components/AboutDataPopover.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import Popover from '@mui/material/Popover'
+import Typography from '@mui/material/Typography'
+import InfoIcon from '@mui/icons-material/Info'
+import { theme } from 'app/theme'
+
+interface AboutDataPopoverProps<T = {}> {
+  content: (props: T) => React.ReactNode
+  props?: T
+  backgroundColor?: string
```

Comment:
I think it would be nice for the about data popovers to have a similar appearance across products. It might be worth checking in with Raj and standardizing on a background.